v2/fleet/sumologic.service

```diff
@@ -0,0 +1,29 @@
+[Unit]
+Description=Sumologic Collector
+After=docker.service
+
+[Service]
+EnvironmentFile=/etc/environment
+User=core
+Restart=always
+ExecStartPre=/usr/bin/docker pull behance/docker-sumologic
+ExecStartPre=-/usr/bin/docker stop sumologic
+ExecStartPre=-/usr/bin/docker rm -f sumologic
+TimeoutStartSec=0
+ExecStartPre=-/usr/bin/docker kill sumologic
+ExecStartPre=-/usr/bin/docker rm sumologic
+ExecStart=/usr/bin/bash -c \
+    "if [[ -f /etc/profile.d/etcdctl.sh ]]; then source /etc/profile.d/etcdctl.sh;fi && \
+    sudo /usr/bin/docker run --name sumologic \
+    -v /var/lib/docker/containers/:/tmp/clogs/:ro \
+    -e SUMO_NAME=docker_cluster \
+    -e SUMO_CATEGORY=be/${NODE_PRODUCT}/${NODE_TIER}/control-logs \
```

Comment:
In the worker tier, this should be `-e SUMO_CATEGORY=be/${NODE_PRODUCT}/${NODE_TIER}/container-logs `